Configurations/unix-Makefile.tmpl

```diff
@@ -1506,11 +1506,12 @@ EOF
       my %args = @_;
       my $lib = platform->staticlib($args{lib});
       my @objs = map { platform->obj($_) } @{$args{objs}};
-      my $objs = join(" \\\n" . ' ' x (length($lib) + 2),
+      my $deps = join(" \\\n" . ' ' x (length($lib) + 2),
                       fill_lines(' ', $COLUMNS - length($lib) - 2, @objs));
+      my $fill_lib = join("\n\t", (map { "\$(AR) \$(ARFLAGS) $lib $_" } @objs));
```

Comment:
For efficiency, better split the list of object file into just a view pieces.

Reply:
@levitte, here is a solution for the efficiency problem, splitting the `ar` call to 250 objects at a time:
*  Restore the original
```
ARFLAGS         => "r",
```
* Replace in your hitherto PR:
```
      my $fill_lib = join("\n\t", (map { "\$(AR) \$(ARFLAGS) $lib $_" } @objs));
```
by
```
      my $max_per_call = 250;
      my @objs_grouped;
      push @objs_grouped, join(" ", splice @objs, 0, $max_per_call) while @objs;
      my $fill_lib = join("\n\t", (map { "\$(AR) \$(ARFLAGS) $lib $_" } @objs_grouped));
```

Reply:
Other than that, sure

Reply:
Done

Reply:
> Not sure I understand why "r" is preferable to "q" ("c" can be appended either way)

I don't know either - I just thought that in doubt the original "r" may be kept.
At least in our case both variants work.